packages/mdl-button/index.scss

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "variables";
+
+.mdl-button {
```

Comment:
I'll experiment with this method a bit and make sure coloring still works fine with it. I don't see why it shouldn't.

For doing ripple-handling later, we just need to make a button JS component that does whatever is needed for ripple interaction, or that just adds a class which overrides the pseudo elements and resets them. That way the natural interactions don't trigger and it is up to JS to drive interaction. Fairly easy progressive-enhancement.